package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "serverless-vpc-plugin",
-  "version": "1.0.6",
+  "name": "serverless-vpc-plugin2",
```

Comment:
We could name it `@compassion-technology/serverless-vpc-plugin`.

Reply:
I did that initially but after making it public I thought it might be odd to leave that in there. I'm ok with anything though

Reply:
I think I've seen that style in published forks of old unmaintained packages before.

Reply:
Oh ok, sounds good